app/views/pages/home.html.erb

```diff
@@ -8,74 +8,45 @@
QKKBef4xRdgVyizEANUqMsTJ6AVXe+OE4zmWbwqNOnTZV1eHTpIpCkxMMCQCbeuCub5dpsWalUq5eofv0XIFvOmToYfIe+EReZa2Uz1c13SuURaLG1FnAbUpAAKTJaRIkMpnDjv6N6DlVM9Vsy5hhOzdC/g7KPwGB3BMi+ZpUqy6Ka1iwp6tTMxRirdKqBuJ+LbDRR5wy5oh28SiCsqayEKfKKgmmb9tU4i5A4zlqOTyWXYjx/Bq1Fn7oaoSR3OpkloAJhD5ibwRNlKtyPnCpNOtSPuoXbvOqpgNxTlDOIwUutXVtpqxMb9LlZiRtO4xp2ZeqysNUz/AGjA7RETN/O3vvgfxvhlGslMVXLMQQpbSYNidNp7Xg30iZwqQWLfDeQ6CqPFLO3cBoX5Qob88F05ayoAAoJbzkn8zfB11sPP9f44gzWZSkjVKhCoglmOwAxjK06LVCbycK6/WvBp0RTbN1yGdmGzBQAiJsAB94YJ8QyGadG1ZlV7kUKWlio+IB3quQSsgMAIMHH3IVNhkqbMCGqF6pHeKlRmX8VKn54K8WYChWJFhTc/gpP4YG9jS0Avoypxw+gY+I1G/Go0flGFD6cj1ZX2qfqmHfk2nUXIZUIU/qlPUD97q7e/mMJH03EzlZiYqTHumHH94Pg88oD+hZb/ACKf7IwYCYC8oN/Qst/k0/2Rg2DiZdBcOGzCKyoWUM0lVJuY3gYUOPn7DisbtURB86NFB+1grzDxDJlDTzFZB36WBdD2YBZIPyvsZGM4bmPRTrZcA1ketTcVBKk6HQkkPLXVAtzaLWxUEKTNoqGCR6441YTF+kbLkS1KsGO4AU/nrE4p5n6R/wDd5f51H/co/ficGVaH+Rj4DyxlOa56zb7MlP8AwIP+/UcB85xivVnxK1RvQsY/AWxSgxZI2PN8Ro0v6ytTT0ZgD+EzgJnuccooszVGG3hqZHszQP1xlSmMSLilBA5Gucvc7UqpIU+FV7K8RUgSLTBMdpkXi2CSc5VyICUgfZj/AN+MKztihNusfKxxYzPEqvhsBWqkR2d4/GcNpk6Nez3M2YHxVhTH/Cv5kT+eFviPNFKPtcyak7AFnn2iRhCCsSAF6mE6ifTeN/574no5AC8kt3JwsQsYa3M9MfBSc/4oX9NWKVTmasfhRFHsSfxJj8sVEoj5+X8PPHfgemCkI8qcSzD71X/4Tp/ZjEH1eTJufM4u0sqzGEVmPkoJPvAk4MZDlXM1BJUUx51DH/TBYfhh2kFNi+lHFimg7/lh0y/Iqj46zE/3FA/MzP4DF/I8oZdJ1hqh7aiRHyUifnhZoeLEjKcPqVf6qmzx3VSQPc7D54OZHlDMEhmZaRHcElh6jTb/AKsO+WoIihUUKo2AEC//AJnEwxDmylEr8Lyj01ipWar/AHmAEQPxPuScJ9DI0c7mM7XaktWiTSp0iRdvCQioUIggTEFTeMFQr8QLKGKZNWKnSerMFTBE/dpyI8zf5Cs1y1mMkC+Trl6c/wBRXgjqN72BEmfuHczi4MUkT1vsMo5pg+CtNlWTf4WEHuQLj5DfcAOFZkplqEHSDSUllgXN7ncmIHyx5xrmMuDSzFFsuzKbG6NMywMXF9xI/vHADh+dBpUwxui6SP8ADafmIvi8mtkqNj3X5oqA0UQQgKFzszAHqHoCJ9cTcz8WSnmFZWVim4uQC0gxBF7T6T7YSvrkXEfM49z3GqeiGB1sJk7bixG0R3Edt8TGb+lTjyjVs7xynSy6VmVmLkKlNbs7t8Kr5kwTPlfALiHLuYza6sxWFEzNOiiipTT/AB6rVX9RYdpxS4fxUVxkqpKjwGd6q3HxUylMqfgXeesrE2w6LxCmQCToB7t8P/OJQ/JjjOT3Y0hTbM8Vy/8AWUaWcpj79I6KnzWIPsqn3xwedMrXSpQqs+WqOjIVrqVI1AiZ2G/eMOp85kb+mKPEcjSrLpq0kqL5OoP4Tt8sLJfUFATkHia1cqlORroAUXCmRKdKkEWKsoBB2N/LCb9OB6sr7VP+zBfiuTo8MqLmKB0gnTXy+uS1NvvKCS0ob7x7XlT+lHjlLNNQNINpUP1NHVq0wRee3fGkV+VifKNF5UqxlMr/AJKfsj+GLPG1rvRYZeoadUXFh1Rup1AgT59jHacUeWP/AImW/wAlP2Rgqu/8+QxEujXDFajEk6pDSZmxBm8jsZ7YrVgRddx/MfhbDn9IuVRcyhCgF6QZj5kGJPrAjCtjVPVkNEFFpHf5jEyqcSDEgwwOFXHYp47XEowgIPDxIq4kx0MAFHOi9P8AxjFniEeE/t+/EGd3p/5i/rhoXhlJuHZisUmoq2aTbqHaY/LCsAHw+ixgAEuQBpAMgAWEb+vvg7k+V8zU/wBloHnUOn8vi/LD1ylTC5TKwANVBGPqSgMnzM3wVbEOWykhOynIg/21Yn0Rf+5p/ZwWp8p5ZRHhlv8AE7z+IIj2j8MGv5/PHwxDky6RU4bwihSLGjT0k7yWJj5k29sXDGPjvjtLwcIDnHjHHzbY5O+AD0+mPRMWie3vjnscdDt/PngAC8so+Wy1KjWRpQEF6YLqZYm2nrG+5X54I1k8ZSaVSm67TvB7yymJHlAxcfEVbIU3Oor1afjUlW9tSkNHpMYLATee8q65ZVKqwAYkGCNzB8xjLqpCmR8x+n8PwxsnDs21bKsasOdZXqVSIFtojtvvjKs9QXSx0gbmwgb9gLD5Y2gyGqKcOTb+fxxZo5IsbsZ8/wDycS5O6j2GGHlTLJUrhXUMPI4bdAtnOV5brqodUaReLSfIr/a2/TfDbwebmilWnU+8oUi/rPRf+9+WC/BMlT0kaQAJhRZb+aix+YwZS0KLACwFgPkMYN2VVAunw3UoNRVR5maRK+xI+Fm33BGFDmKhxNZHiPUpjY0YBI/vBAHnz3GH848ff8MNOgZhDNBM7zf37zhd4sNJCfdBJX0DQY+Rx+heIcJo5gkVqavbfZv+YQ354/PnHbVmHYE42i7M5Kj/2Q=="
```

Comment:
Storing dynamic images is always tricky. Ideally you want to use a specialized service such as s3 or cloudinary, but you could also use active storage to save them in your db: https://levelup.gitconnected.com/rails-image-upload-101-f9bf245e389b.

Reply:
Thank you, I will do this. After installing it seems like it just a matter of adding to the image model.